lib/expansion_tile_card.dart

```diff
@@ -95,64 +102,77 @@ class ExpansionTileCard extends StatefulWidget {
   /// Defaults to 2.0.
   final double elevation;
 
+  ///The elevetion when collapsed
+  ///
+  ///Defaults to 0.0
+  final double initialElevation;
+
+  final Color shadowColor;
+
   /// Specifies if the list tile is initially expanded (true) or collapsed (false, the default).
   final bool initiallyExpanded;
 
   /// The padding around the outside of the ExpansionTileCard while collapsed.
-  /// 
+  ///
   /// Defaults to EdgeInsets.zero.
   final EdgeInsetsGeometry initialPadding;
 
   /// The padding around the outside of the ExpansionTileCard while collapsed.
-  /// 
+  ///
   /// Defaults to 6.0 vertical padding.
   final EdgeInsetsGeometry finalPadding;
 
   /// The inner `contentPadding` of the ListTile widget.
-  /// 
+  ///
   /// If null, ListTile defaults to 16.0 horizontal padding.
   final EdgeInsetsGeometry contentPadding;
 
   /// The background color of the unexpanded tile.
-  /// 
+  ///
   /// If null, defaults to Theme.of(context).canvasColor.
   final Color baseColor;
 
   /// The background color of the expanded card.
-  /// 
+  ///
   /// If null, defaults to Theme.of(context).cardColor.
   final Color expandedColor;
 
   /// The duration of the expand and collapse animations.
-  /// 
+  ///
   /// Defaults to 200 milliseconds.
   final Duration duration;
 
   /// The animation curve used to control the elevation of the expanded card.
-  /// 
+  ///
   /// Defaults to Curves.easeOut.
   final Curve elevationCurve;
 
   /// The animation curve used to control the height of the expanding/collapsing card.
-  /// 
+  ///
   /// Defaults to Curves.easeIn.
   final Curve heightFactorCurve;
 
   /// The animation curve used to control the rotation of the `trailing` widget.
-  /// 
+  ///
   /// Defaults to Curves.easeIn.
   final Curve turnsCurve;
 
   /// The animation curve used to control the header, icon, and material colors.
-  /// 
+  ///
   /// Defaults to Curves.easeIn.
   final Curve colorCurve;
 
   /// The animation curve used by the expanding/collapsing padding.
-  /// 
+  ///
   /// Defaults to Curves.easeIn.
   final Curve paddingCurve;
 
+  ///This color will overwrite all other colors
+  final Color color;
+
```

Comment:
I'm not sure this is a good idea, or at the very least not with this name. At a minimum this should be renamed `overrideColor`, but I'm not sure why it's needed.

This job can already be done by setting `baseColor` and `expandedColor` to be the same, only difference being (possibly?) performance as you're skipping the animation.

Reply:
The reason I did this was because the color wouldn't change at all when I tried setting baseColor and expandedColor to the same thing. You are probably right that this is unnecessary and I should try to find the underlying cause instead.

Reply:
Fair enough. I had a great deal of difficulty getting the colors to work as I expected them to. The issue may be something on Flutter's end. I'm fine accepting this with the name `overrideColor`

Reply:
Or `constantColor` to match elevation

Reply:
So I figured this out... I didn't realize that the changes weren't applied until I restarted the app since they are set in initState. I have reverted this commit, as well as the one that added the constantElevation.